lib/aspect.ts

```diff
@@ -0,0 +1,32 @@
+import { IAspect, IConstruct, App, Stack } from "@aws-cdk/core";
+import { Watchful } from 'cdk-watchful';
+import * as sqs from '@aws-cdk/aws-sqs';
+import * as ddb from '@aws-cdk/aws-dynamodb';
+
+export class WatchAspect implements IAspect {
```

Comment:
Are you also applying the aspect anywhere?

Reply:
And what does this look like if you are deploying multiple application stages? Beta and Prod?